### PR TITLE
Completed dom-event-delegation exercise. - Jeff Jin

### PR DESCRIPTION
**Pull request by jeffmjin**
https://user-images.githubusercontent.com/110066310/189193782-66df0820-cf32-472d-be58-7c32b29ab338.mov

https://user-images.githubusercontent.com/110066310/189193791-176833a3-b5dc-4d8d-b300-55d151591c63.mov

Updated**

https://user-images.githubusercontent.com/110066310/189506285-88e09ca9-ff29-4ed4-98e2-20f3a8642cec.mov

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var task = document.querySelector('.task-list');
+task.addEventListener('click', handleClick);
+
+var list = document.querySelector('.task-list-item');
```
Comment (BrettAlbright) [inline dom-event-delegation/main.js]: you don't want to query this here.

File: dom-event-delegation/main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+function handleClick(event) {
+  console.log(event.target);
+  console.log(event.target.tagName);
+  if (event.target && event.target.matches('button')) {
+    list.remove();
```
Comment (BrettAlbright) [inline dom-event-delegation/main.js]: you want to remove the ancestor which matches the `.task-list-item` selector of the button which was clicked. 
you don't just want to remove the first one which was selected in the list variable

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Jeff, you want to make sure that you are finding the task list item ancestor of the specific button you are clicking on. Let me know if you have any questions on that!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**